tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/sampleNeutrino/1/NeutrinoInput/Measurements/results.csv

```diff
@@ -0,0 +1,301 @@
+0.0292761,0.0266502

```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file